.github/workflows/rust.yml

```diff
@@ -0,0 +1,59 @@
+name: Rust
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - 'sdk/rust/**'
+      - '!sdk/rust/README.md'
+      - 'resources/porcupine/lib/beaglebone/**'
```

Comment:
does this work with the submodules? it may be the case that we can only detect whether a submodule has been updated or not